scripts/test.mjs

```diff
@@ -14,7 +14,6 @@ const root = path.dirname(dirname)
 const force = Bare.argv.includes('--force-install')
 
 const dirs = [
-  path.join(root, 'test', 'fixtures', 'harness', 'node_modules'),
   path.join(root, 'test', 'fixtures', 'encrypted', 'node_modules'),
```

Comment:
can we also remove this line to npm install for encrypted since encrypted has no deps now